README.md

```
@@ -54,6 +54,7 @@ A list of remote job boards.
 | ❇️ | [Remote.co](https://remote.co/remote-jobs/) | A remote jobs board. |🌟|
 | ❇️ | [Remote.com](https://remote.com/jobs) | A remote jobs board. |🌟|
 | ❇️ | [Remote.jobs](https://remote.jobs/) | A remote jobs board. |🌟|
+| ❇️ | [Remote Job Assistant](https://remotejobassistant.com/) | Remote jobs for non-technical roles. |🌟|
```

markdownlint-cli2 (0.18.1)

Table column count
Expected: 5; Actual: 4; Too few cells, row will be missing data

(MD056, table-column-count)